packages/react-components/src/organisms/ProfileRecentWorks.tsx

```diff
@@ -55,49 +59,65 @@ const typeMap: { [key in OrcidWork['type']]: string } = {
 };
 
 const containerStyles = css({
-  display: 'flex',
-  flexDirection: 'row',
-  flexWrap: 'wrap',
-  marginLeft: '-8px',
```

Comment:
You're lucky I never came across this :axe:

Reply:
If it works it ain't stupid.